compiler/rustc_mir_transform/src/coroutine.rs

```diff
@@ -1331,7 +1393,8 @@ fn create_coroutine_resume_function<'tcx>(
 
     if can_return {
         let block = match coroutine_kind {
-            CoroutineKind::Async(_) | CoroutineKind::Coroutine => {
+            // FIXME(gen_blocks): Should `async gen` yield `None` when resumed once again?
```

Comment:
I tend to think no, but I can see arguments either way. Something to figure out for the RFC, I guess.